test/e2e/ems/ems_test.go

```diff
@@ -13,8 +13,7 @@ import (
 	"github.com/elastic/cloud-on-k8s/test/e2e/test/maps"
 )
 
-// TestElasticMapsServerCrossNSAssociation tests associating Elasticsearch and Elastic Maps Server running in different namespaces.
-func TestElasticMapsServerCrossNSAssociation(t *testing.T) {
+func maybeSkip(t *testing.T) {
```

Comment:
Should we manage this logic in https://github.com/elastic/cloud-on-k8s/blob/f81f51fb6e5eb39882454396b8f2f83227ffa10b/test/e2e/test/maps/builder.go#L150-L153 ?

Reply:
Excellent idea. I forgot about that feature of the builder! I think all of the these checks should go there. The 7.11.0 check is actually redundant.

Reply:
Yes! I also completely forgot the existence of this method.